### DISCUSSION TITLE
Night does not pass with "No amount of sleep will pass this night" message

### DISCUSSION THREAD

**Discussion by mswmaurer** [Nuytemans-Dieter/BetterSleeping#278] [General]
These seems to happen every few nights.  I can get out of bed and get back in to pass the night.  I can't find this message in any of the config files and can't find a setting that might be causing this.  Any insights?

**Comment by mswmaurer** [comment]
Turns out this is a MInecraft server message.  It appears Minecraft may have added the ability to have only a portion of the players sleep?

**Comment by mswmaurer** [comment]
It appears that BetterSleeping "might" be impacting the playersSleepingPercentage GameRule.  I set the GameRule to 30, then added BetterSleeping to my server and restarted the world to load it.  The GameRule is supposed to persist with the world, but is now set to "200," which could explain the occasional "night won't pass" problem.  I assume this is being done to override the default behavior with the plug-in, but there seems to be something not always working.

**Comment by Nuytemans-Dieter** [comment]
Thank you for reporting this issue! I'll take a look at it soon!
Made an issue for this: #279
